qa_forum/static/js/main.js

```diff
@@ -0,0 +1,39 @@
+$(document).ready(function(){
+
+    $('.comment_submit').on('click', function() {
+        var raiser = $(this).prev();
+        var comment = $(this).parent().prev().children();
+        var pointTo = $(this).val();
+        if (raiser.val() && comment.val()) {
+            send_comment(raiser, comment, pointTo);
```

Comment:
Personally I use snake case because of the habit in Python. I know camel case is used more often in Javascript.